internal/backend/integrations/integrations.go

```diff
@@ -59,6 +61,7 @@ func Start(_ context.Context, l *zap.Logger, mux *http.ServeMux, vars sdkservice
 	github.Start(l, mux, vars, o, d)
 	google.Start(l, mux, o, d)
 	httpint.Start(l, mux, d, c, p)
+	jira.Start(l, mux, vars, o, d)
```

Comment:
Nit. We start web server per integration, even if there's no connection using it. Can we make it lazy?